mobile/lib/models/location_history.dart

```diff
@@ -78,6 +78,13 @@ class LocationHistory extends Equatable {
 
   Map<String, dynamic> toJson() => _$LocationHistoryToJson(this);
 
+  Map<String, dynamic> toAPiJson(String firebaseUserId) {
```

Comment:
We can rename this function to `toAPIJson` @BenjaminSsempala

Reply:
Which function @NoahNsimbe

Reply:
`toAPiJson` to `toAPIJson`